src/pages/ShiftPlan/index.js

```diff
@@ -43,16 +54,66 @@ const ShiftPlan = () => {
       .catch((error) => message.error('An error occurred: ' + error.message));
   };
 
+  const filterEventsWithUser = (events, userId) => {
+    var eventidsWithUser = engagementState.engagements
+      .filter((engagement) => engagement.userId === userId)
+      .map((e) => e.eventId);
+    return events.filter((_event) => eventidsWithUser.includes(_event.id));
+  };
+
   return (
     <SideBarPage title="Tender site">
+      <Collapse
```

Comment:
Ser mega godt ud med filterne, men jeg vil anbefale at de bliver factored ud til `/styles/molecules/shiftfilter/index.js`.

Reply:
Har lavet nogle andre conditions siden det nok ikke er relevant for members anyway at se bagud så nu bliver der filteret på egne shifts og dem som er up for grabs